src/Http/Controllers/Swagger/FileApiSwagger.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+namespace EscolaLms\Files\Http\Controllers\Swagger;
+
+use EscolaLms\Files\Http\Requests\FileDeleteRequest;
+use EscolaLms\Files\Http\Requests\FileListingRequest;
+use EscolaLms\Files\Http\Requests\FileMoveRequest;
+use EscolaLms\Files\Http\Requests\FileUploadRequest;
+use Illuminate\Http\JsonResponse;
+
+/**
+ * @OA\Info(title="Files API", version="1.0.0-dev")
```

Comment:
@OA\Info in package blocks generation of OpenApi/Swagger docs in EscolaLMS/API if we try to include packages in documentation, because there can only one @OA\Info in whole project

Reply:
And make sure that every route has some tags, for example `tags={"Files"}` for correct grouping

Reply:
remove `@OA\Info(title="Files API", version="1.0.0-dev")` 

add `SWAGGER_VERSION` string, eg https://github.com/EscolaLMS/H5P/blob/main/src/HeadlessH5PServiceProvider.php#L31 

then replace it in CI with php or sed or something, eg https://github.com/EscolaLMS/H5P/blob/main/.github/workflows/swagger.yml#L23

otherwise this will block Swagger generation in [API](https://github.com/EscolaLMS/API) for all endpoints from all packages